docs/rst/_templates/footer.html

```diff
@@ -0,0 +1,11 @@
+{% extends "!footer.html" %}
+
+{% block extrafooter %}
+{{ super() }}
+<div class="footer"><p>Please see
+    our <a href="https://www.mdanalysis.org/pages/privacy/">Privacy Policy</a>
+    to learn how <a href="https://www.mdanalysis.org">MDAnalysis</a> collects data.</p>
+    <script data-goatcounter="https://mdanalysis.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
```

Comment:
Note that this part is necessary for analytics.

The number of page accesses is actually an important number for us to demonstrate impact so we do want to keep track of which pages are seen. This is fully anonymous, though.